version-reference

TanyaStere42 opened the pull request:
version can be foundin the footer and actually updates, so it's worth calling out

Atticus29 commented on docs/researchers/codex/introduction.md:
```diff
@@ -5,8 +5,19 @@ title: An Introduction to Codex
+* A high performance PostgreSQL database for storing multiple wildlife-related data types
+* Two servers:
+  * Data Management Server - Coordinates UI browser display and data storage
+  * Wildbook Image Analysis ("WBIA") - Manages the computer vision pipeline and and artificial intelligence (AI) models and training tools. WBIA does not come pre-configured to detect and/or identify animals from a species. Pre-trained machine learning (ML) models for the species must be used, or new models must be created and configured using training data.
```
I'm not clear whether this should be WBIA or SAGE. It feels weird to have the word "Wildbook" in one of Codex's features.

TanyaStere42 replied:
WBIA until we get the branding done on that platform

Atticus29 reviewed (commented):
I think "disperse" is an actual error.
The other I leave to your discretion.

TanyaStere42 commented on docs/researchers/codex/introduction.md:
```diff
@@ -5,8 +5,19 @@ title: An Introduction to Codex
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-This topic provides an overview of Codex.
+Codex is a web-based, multi-user software platform designed to help researchers collaboratively track individual animals in wildlife populations and estimate population size. Each Codex installation can be used by multiple researchers and organizations to study multiple species across disperse or narrow regions. Codex is also valuable as a catalog reconciliation tool.
```
went with diverse, although I think I did it wrong when I committed directly to the branch??

Atticus29 replied:
I think that was what you were supposed to do!

Atticus29 reviewed (approved):
LGTM!

TanyaStere42 committed:
version-reference

version can be foundin the footer and actually updates, so it's worth calling out

goddesswarship reviewed (approved):
👍
